src/lichess/interfaces/tournament.ts

```diff
@@ -158,11 +158,13 @@ export interface TournamentLists {
 }
 
 export interface TournamentListItem {
+  readonly battle?: {}
```

Comment:
why empty object as type? the best would be to type it, or you can use `any` for now.